pkg/daemon/ceph/osd/daemon.go

```diff
@@ -328,6 +328,10 @@ func getAvailableDevices(context *clusterd.Context, agent *OsdAgent) (*DeviceOsd
 				logger.Errorf("failed to get udev info of partition %q. %v", device.Name, err)
 				continue
 			}
+			if device.PartitionTableType != sys.GPTType {
+				logger.Infof("skipping partition %q because it is not a gpt partition", device.Name)
```

Comment:
How about logging the partitionTableType detected?

Reply:
I can do that, but just so you are aware, `udev` doesn't detect atari partitions, and this field will be an empty string.